quickstart/Readme.md

```diff
@@ -1,4 +1,4 @@
 # Quick Start
 
-For the documentation of this tutorial, please see the Fluence [developer documentation](https://doc.fluence.dev/docs/quick-start).
+For the documentation of this tutorial, please see the Fluence [developer documentation](https://fluence.dev/docs/build/quick-start/).
 If you have any questions, please contact us in our[discord]( https://fluence.chat) channel, raise an issue or open a PR.
```

Comment:
not a functional issue but  that extra space  in ( https://fluence.chat) probably should go.

Reply:
fixed